### PR TITLE
MESH-1234: Allow GC to issue & revoke a CDD claim 

### PR DESCRIPTION
**Pull request by Centril**

### CODE REVIEW

File: pallets/identity/src/lib.rs [hunk 1]
```diff
@@ -633,6 +633,24 @@ decl_module! {
...
 
+        /// Assuming this is executed by the GC voting majority, adds a new cdd claim record.
+        #[weight = 950_000_000]
+        pub fn gc_add_cdd_claim(
+            origin,
+            target: IdentityId,
+            expiry: Option<T::Moment>,
+        ) -> DispatchResult {
+            T::GCVotingMajorityOrigin::ensure_origin(origin)?;
+            Self::base_add_cdd_claim(target, Claim::CustomerDueDiligence([0; 32].into()), GC_DID, expiry)
```
Comment (vkomenda) [inline pallets/identity/src/lib.rs:644]: If this is supposed to be a wildcard CDD claim, perhaps it's better to use the existing function `Claim::make_cdd_wildcard` to construct it.

Comment (Centril) [reply pallets/identity/src/lib.rs]: I've applied your suggestion; thanks for the tip!

**Review by vkomenda — APPROVED** [review]

File: pallets/identity/src/lib.rs [hunk 2]
```diff
@@ -633,6 +633,24 @@ decl_module! {
...
+            T::GCVotingMajorityOrigin::ensure_origin(origin)?;
+            Self::base_add_cdd_claim(target, Claim::make_cdd_wildcard(), GC_DID, expiry)
+        }
+
+        /// Assuming this is executed by the GC voting majority, removes an existing cdd claim record.
+        #[weight = 500_000_000]
+        pub fn gc_revoke_cdd_claim(origin, target: IdentityId) {
+            T::GCVotingMajorityOrigin::ensure_origin(origin)?;
+            Self::base_revoke_claim(target, ClaimType::CustomerDueDiligence, GC_DID, None)
+        }
```
Comment (satyamakgec) [inline pallets/identity/src/lib.rs:652]: It is better to add last in the list of dispatchable.

Comment (Centril) [reply pallets/identity/src/lib.rs]: Thanks, fixed.

**Review by maxsam4 — APPROVED** [review]

**Review by satyamakgec — APPROVED** [review]

**Pull request merged**